.circleci/config.yml

```diff
@@ -30,26 +30,87 @@ workflows:
           post-steps:
             - codecov/upload
             - run:
-                command: |
-                  npm run lint
+                command: npm run lint
                 name: Lint
             - run:
-                command: |
-                  npm audit
+                command: npm audit
                 name: Audit
 
       - cypress/run:
-          name: e2e
+          name: e2e chromium@101.0.4951.64
+          executor: cypress
+          post-checkout:
+            - run:
+                name: Install wget/unzip
+                command: |
+                  apt update -y
+                  apt-get install -y wget
+                  apt-get install -y unzip
+            - run:
+                name: Download/Install Chromium
+                command: |
+                  wget -O chrome-linux.zip "https://www.googleapis.com/download/storage/v1/b/chromium-browser-snapshots/o/Linux_x64%2F982481%2Fchrome-linux.zip?generation=1647559751806495&alt=media"
+                  unzip chrome-linux.zip
+          start: npm start
+          command: npx cypress run --browser chrome-linux/chrome
+          no-workspace: true
+          wait-on: http://localhost:4200
+          store_artifacts: true
+
+      - cypress/run:
+          name: e2e chromium@102.0.5005.61
           executor: cypress
+          post-checkout:
+            - run:
+                name: Install wget/unzip
+                command: |
+                  apt update -y
+                  apt-get install -y wget
+                  apt-get install -y unzip
+            - run:
+                name: Download/Install Chromium
+                command: |
+                  wget -O chrome-linux.zip "https://www.googleapis.com/download/storage/v1/b/chromium-browser-snapshots/o/Linux_x64%2F992740%2Fchrome-linux.zip?generation=1649975445134595&alt=media"
+                  unzip chrome-linux.zip
+          start: npm start
+          command: npx cypress run --browser chrome-linux/chrome
+          no-workspace: true
+          wait-on: http://localhost:4200
+          store_artifacts: true
+
+      - cypress/run:
+          name: e2e chrome/edge@latest
```

Comment:
I see `edge` here, but the description also mentions `electron`. Are we including that browser as well?

Reply:
Cypress includes electron by default, so no installation necessary.